core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockStoreMeta.java

```diff
@@ -61,13 +61,23 @@
 
   @Override
   public Map<String, List<Long>> getBlockList() {
```

Comment:
@yabola it's always to put the comment in the getBlockList() method declaration. That's where people will usually view to understand the method. And the comment should be implementation independent and as a contract between caller and implementor.
For example, you can change the BlockStoreMeta.getBlockList description:
```
This method is only available when users initialize the blockStoreMeta with full block meta requirement
```

Reply:
done